UnitTests/BraintreePayPalTests/BTPayPalVaultRequest_Tests.swift

```diff
@@ -65,4 +65,13 @@ class BTPayPalVaultRequest_Tests: XCTestCase {
         XCTAssertEqual(shippingParams["country_code"], "US")
         XCTAssertEqual(shippingParams["recipient_name"], "Recipient")
     }
+    
+    func testParametersWithConfiguration_whenUserPhoneNumberIsPresent_SetsPayerPhoneNumberInRequest() {
```

Comment:
Nit - you can probably lump this assertion for phone number within the JSON into the test above

Reply:
addressed in d40404527b3b7ffc6a5d7e535ce6491f39525eea !